### PR TITLE
feat: kurtosis package init command

### LINKED ISSUE #1416: FR: add new `kurtosis package init` command to simplify package creation
### Background & motivation

Part of the Forked Packages Cleanup project. This is a new command to make it way easier for a developer to go from starlark to running package. 

Implementation details: https://www.notion.so/kurtosistech/2023-09-Forked-Packages-Cleanup-d8bc4f5165ae4b039c07cdad56892459?pvs=4#d8559c49bc644067ab32049f022ff196

### Desired behaviour

This command initializes the current directory to be a Kurtosis package by creating a `kurtosis.yml` file with the given package name automatically for me.

### How important is this to you?

Nice to have; this feature would make using Kurtosis more enjoyable.

### PR DESCRIPTION
**Pull request by leoporoli**
## Description:
kurtosis package init command

## Is this change user facing?
YES

## References (if applicable):
Fix #1416 

This is part of the [Forked cleanup package project](https://www.notion.so/kurtosistech/Forked-Package-Cleanup-16d86c4e274547b28496f17154bf3d62)

### CODE REVIEW

File: cli/cli/kurtosis_package/kurtosis_package_test.go [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+		PackageReplaceOptions: map[string]string{},
+	}
+
+	err = yaml.UnmarshalStrict(fileBytes, kurtosisYamlObj)
+	require.NoError(t, err)
+
+	require.Equal(t, packageName, kurtosisYamlObj.PackageName)
+}
+
+func TestInitializeKurtosisPackage_FailsIfMainStarFileAlreadyExist(t *testing.T) {
```
Comment (h4ck3rk3y) [inline cli/cli/kurtosis_package/kurtosis_package_test.go:113]: what if a person is trying to make a script a package?

Comment (h4ck3rk3y) [reply cli/cli/kurtosis_package/kurtosis_package_test.go:113]: based on my read of the code it should work, but can we test that?

File: cli/cli/kurtosis_package/kurtosis_package.go [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+	"os"
+	"path"
+)
+
+const (
+	kurtosisPackageFilePermissions os.FileMode = 0644
+	kurtosisYmlFilename                        = "kurtosis.yml"
+	mainStarFilename                           = "main.star"
+	mainStarFileContentStr                     = `def run(plan):
+    # TODO
```
Comment (h4ck3rk3y) [inline cli/cli/kurtosis_package/kurtosis_package.go:19]: can we do ` plan.print("hello world!") instead

1. they get something functional
2. they can see how `plan` works

**Review by h4ck3rk3y — APPROVED** [review]

File: cli/cli/kurtosis_package/kurtosis_package.go (hunk 2 above)
Comment (leoporoli) [reply cli/cli/kurtosis_package/kurtosis_package.go:19]: good suggestion. I'll do it

File: cli/cli/kurtosis_package/kurtosis_package_test.go (hunk 1 above)
Comment (leoporoli) [reply cli/cli/kurtosis_package/kurtosis_package_test.go:113]: done

**Pull request merged**